be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -165,9 +166,20 @@ class IndexChannel {
 
     void set_tablets_received_rows(
             const std::vector<std::pair<int64_t, int64_t>>& tablets_received_rows, int64_t node_id);
+    void set_tablets_filtered_rows(
+            const std::vector<std::pair<int64_t, int64_t>>& tablets_filtered_rows, int64_t node_id);
+    int64_t num_rows_filtered() {
```

Comment:
it seems not a complete function define. how to pass the compile?

Reply:
which one